cmd/main.go

```diff
@@ -71,8 +71,11 @@ func main() {
 	if *enableProfile {
 		klog.InfoS("Starting profiling", "port", *profilePort)
 		go func() {
-			addr := fmt.Sprintf("%s:%d", "localhost", *profilePort)
-			klog.ErrorS(http.ListenAndServe(addr, nil), "unable to start profiling server")
+			server := &http.Server{
+				Addr:              fmt.Sprintf("%s:%d", "localhost", *profilePort),
+				ReadHeaderTimeout: 5 * time.Second,
```

Comment:
`G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)`

Reply:
should we make this a const?

Reply:
done!